version/1/3/0/pl/code_of_conduct.md

```diff
@@ -0,0 +1,24 @@
+# Kodeks postępowania współtwórców
+
+My, współtwórcy, działając dla dobra oraz otwartości naszej społeczności, ślubujemy poszanowanie dla wszystkich osób które wnoszą wartość do naszej pracy. Wartość tą wnoszą między innymi poprzez: zgłaszanie problemów, propozycji zmian, aktualizację dokumentacji, czy zgłaszanie poprawek poprzez "pull request".
+
+Jesteśmy zdeterminowani, by współpraca nad tym projektem była dla wszystkich wolna od zastraszania niezależnie od: doświadczenia; płci; identyfikacji, ekspresji i orientacji seksualnej; kalectwa; wyglądu; rozmiaru ciała; rasy; przynależności etnicznej; wieku; wyznawanej religii ani narodowości.
```

Comment:
(in polish)

"poszczególne punkty wyliczenia zamyka się jakimś znakiem interpunkcyjnym. Może to być przecinek (gdy punkty są krótkie), _średnik (gdy są dłuższe i same zawierają przecinki) albo kropka (gdy punkty mają charakter pełnych zdań)_."
more: http://sjp.pwn.pl/poradnia/haslo/interpunkcja-w-wyliczeniach;1536.html